### PR TITLE
Change search value to a case insensitive check

### PR DESCRIPTION
**Pull request by clicktravel-toddrunham**

### CODE REVIEW

File: lib/components/Form/AutoComplete/index.jsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ const AutoComplete = ({ items, onClick, searchValue }) => (
   <ul className="autocomplete">
     {
       items.map((item) => {
-        const formattedText = { __html: item.text.replace(searchValue, `<b>${searchValue}</b>`) };
+        const formattedText = { __html: item.text.replace(new RegExp(searchValue, 'ig'), `<b>${searchValue}</b>`) };
```
Comment (clicktravel-jan) [inline lib/components/Form/AutoComplete/index.jsx]: Elaborate

File: lib/components/Form/form.stories.js [hunk 2]
```diff
@@ -90,7 +90,7 @@ stories.add('Form Group', () => (
 stories.add('Auto Complete', () => (
   <UI>
     <Component>
-      <AutoComplete onChange={() => { }} items={[{ text: 'item value 1' }, { text: 'item value 2' }]} searchValue="item" />
+      <AutoComplete onChange={() => { }} items={[{ text: 'item value 1' }, { text: 'item value 2' }]} searchValue="Item" />
```
Comment (clicktravel-jan) [inline lib/components/Form/form.stories.js]: Should we change this one to be upper case or the declaration to be lower case?

File: lib/components/Form/AutoComplete/index.jsx (hunk 1 above)
Comment (clicktravel-toddrunham) [reply lib/components/Form/AutoComplete/index.jsx]: In regex i = case insensitive mode and g = global search I.E replace all instances

File: lib/components/Form/form.stories.js (hunk 2 above)
Comment (clicktravel-toddrunham) [reply lib/components/Form/form.stories.js]: Either way works as a test

**Pull request merged**